guide/src/examples/TabsEx.js

```diff
@@ -20,31 +21,43 @@ export default React.createClass({
     render() {
         return (
             <div>
-                <Tabs
-                    tabList={[
-                        "Red Fish",
-                        "Blue Fish",
-                        "Green Fish",
-                    ]}
-                    current={this.state.currentTab}
-                    color={ theme.color.primary }
-                    onTabClick={ this.onTabClick }
-                />
+                <Paper 
```

Comment:
While I was refactoring Tab styles I remembered we don't want to style the background as there are different contexts in which one might need to display it.